ethminer/MinerAux.h

```diff
@@ -574,9 +576,11 @@ class MinerCLI
 	{
 		_out
 			<< "Work farming mode:" << endl
+#if ETH_GETWORK
```

Comment:
Should the farm and stratum parameters be made mutually exclusive?

Reply:
I would prefer something like this. Most stuff already is.
Also i would like to change all the argument parsing and handling to ``cxxopts``